src/Chapter3.hs

```diff
@@ -970,6 +1058,21 @@ implement the following functions:
 
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
+data WeekDay = Sunday|Monday|Tuesday|Wednesday|Thursday|Friday|Saturday deriving (Show,Read,Eq,Ord,Enum)
+
+isWeekend::WeekDay->Bool
+isWeekend d 
+  | d == Sunday || d == Saturday = True
+  | otherwise = False
```

Comment:
Operators `==` and `||` already return a value of type `Bool`, so you can avoid using guards 💂🏻 
```suggestion
isWeekend d = d == Sunday || d == Saturday
```